src/module/DiffusionSDE.cpp

```diff
@@ -183,23 +219,26 @@ void DiffusionSDE::process(Candidate *candidate) const {
 	}
 
 	candidate->setNextStep(nextStep);
-	//candidate->setNextStep(maxStep);
-    // Debugging and Testing
-    // Delete comments if additional information should be stored in candidate
     
+    	// Debugging and Testing
```

Comment:
I also thought about this but wasn't sure how to implement it. If you do have an idea just let me know or do it on your own.

Reply:
I would leave these debugging details for later since we still need to design and prescribe guidelines how to use it. After we define this we will go through the code to refactor it.